quality_control_oca/views/qc_inspection_view.xml

```diff
@@ -0,0 +1,337 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2010 NaN Projectes de Programari Lliure, S.L.
+    Copyright 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+    Copyright 2014 Oihane Crucelaegui - AvanzOSC
+    Copyright 2017-2020 ForgeFlow S.L.
+    Copyright 2017 Simone Rubino - Agile Business Group
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="qc_inspection_form_view">
+        <field name="name">qc.inspection.form</field>
+        <field name="model">qc.inspection</field>
+        <field name="arch" type="xml">
+            <form string="Inspection">
+                <header>
+                    <button
+                        name="action_draft"
+                        type="object"
+                        attrs="{'invisible': [('state', '!=', 'canceled')]}"
+                        string="Draft"
+                    />
+                    <button
+                        name="action_todo"
+                        type="object"
+                        class="oe_highlight"
+                        states="draft"
+                        string="Mark todo"
+                    />
+                    <button
+                        name="action_confirm"
+                        type="object"
+                        class="oe_highlight"
+                        states="ready"
+                        string="Confirm"
+                        icon="fa-check"
+                    />
+                    <button
+                        name="action_approve"
+                        type="object"
+                        states="waiting"
+                        class="oe_highlight"
+                        groups="quality_control_oca.group_quality_control_manager"
+                        string="Approve"
+                    />
+                    <button
+                        name="action_cancel"
+                        type="object"
+                        attrs="{'invisible': [('state', 'not in', ['waiting', 'ready', 'failed', 'success'])]}"
+                        string="Cancel"
+                        icon="fa-ban"
+                    />
+                    <field
+                        name="state"
+                        widget="statusbar"
+                        statusbar_visible="draft,waiting,success"
+                        statusbar_colors='{"success": "blue", "failed": "red"}'
```

Comment:
Use `decoration-{$name}` instead

Reply:
In v12 the colors were not applied to the status bar yet. So I choosed to delete this for keep the odoo style.

Reply:
I'm sorry @CarlosRoca13, I was confused. But yes, I agree to remove it, I think it's an attribute used in old Odoo versions and now it doesn't work, it no longer exists (neither in v12 nor in v13)